### DISCUSSION TITLE
Tags in Neuracache style notes

### DISCUSSION THREAD

**Discussion by pestkaonline** [ObsidianToAnki/Obsidian_to_Anki#188] [Q&A]
Hi,
Is there any way to add tags to notes created in Neuracache style? I mean something like

```
In Neuracache style, to make a flashcard you do #flashcard
The next lines then become the back of the flashcard
Tags: Tag1 Tag2 Tag3
```

**Comment by pestkaonline** [comment]
Unfortunately alternative is using
https://github.com/reuseman/flashcards-obsidian

**Reply by tathagatagupta** [reply]
I found out that you have to use "Tags: tag1, etc" in the same line as the last line. Do not go to a new line for tags. So an example is this: " ... some text in the last line. Tags: tag1" This will work not just with Neuracache but with all styles.
